### PR TITLE
Finish install Devise

### PR DESCRIPTION
**Pull request by trungnn142**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <%= link_to t("static_page.logo"), root_path, id: "logo" %>
     <nav>
       <ul class="nav navbar-nav navbar-right">
+        <% if user_signed_in? %>
         <li><%= link_to t("user.index"), " " %></li>
```
Comment (vinhnd2010) [inline app/views/layouts/_header.html.erb]: indent

Comment (vinhnd2010) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -3,20 +3,24 @@
...
+              <li><%= link_to t("user.profile"), "" %> </li>
+              <li><%= link_to t("user.setting"), "" %></li>
+              <li class="divider"> </li>
+              <li>
+                <%= link_to t("user.logout"), destroy_user_session_path, method: "delete" %>
+              </li>
+            </ul>
+          </li>
+        <% else %>
+          <li><%= link_to t("user.login"), user_session_path %></li>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: new_user_session_path

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -3,20 +3,24 @@
...
-            <li><%= link_to t("user.profile"), "" %> </li>
-            <li><%= link_to t("user.setting"), "" %></li>
-            <li class= "divider"> </li>
-            <li>
-              <%= link_to t("user.logout"), "", method: "delete" %>
-            </li>
-          </ul>
-        </li>
+        <% if user_signed_in? %>
+          <li><%= link_to t("user.index"), " " %></li>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: remove space in " "

File: config/initializers/devise.rb [hunk 4]
```diff
@@ -0,0 +1,265 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
```
Comment (ThuBM) [inline config/initializers/devise.rb]: remove all commented lines in this file

File: config/initializers/kaminari_config.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  config.default_per_page = 10
+  # config.max_per_page = nil
```
Comment (ThuBM) [inline config/initializers/kaminari_config.rb]: remove all commented files in this file

File: config/initializers/kaminari_config.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  config.default_per_page = 10
```
Comment (ThuBM) [inline config/initializers/kaminari_config.rb]: 10 -> Settings

File: config/initializers/kaminari_config.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+  # config.max_per_page = nil
+  config.window = 2
```
Comment (ThuBM) [inline config/initializers/kaminari_config.rb]: 2 -> Settings

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -3,20 +3,24 @@
...
-            <li>
-              <%= link_to t("user.logout"), "", method: "delete" %>
-            </li>
-          </ul>
-        </li>
+        <% if user_signed_in? %>
+          <li><%= link_to t("user.index"), "" %></li>
+          <li class="dropdown">
+            <a href="" class="dropdown-toggle" data-toggle="dropdown">
+              <%= t("user.account") %><b class="caret"></b>
```
Comment (HieuMeo) [inline app/views/layouts/_header.html.erb]: Remove ()

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -3,20 +3,24 @@
...
+            </a>
+            <ul class="dropdown-menu">
+              <li><%= link_to t("user.profile"), "" %> </li>
+              <li><%= link_to t("user.setting"), "" %></li>
+              <li class="divider"> </li>
+              <li>
+                <%= link_to t("user.logout"), destroy_user_session_path, method: "delete" %>
```
Comment (HieuMeo) [inline app/views/layouts/_header.html.erb]: method: :delete

**Comment by ThuBM** [comment]
ok

**Pull request merged**